libraries/AP_Notify/Display.cpp

```diff
@@ -526,8 +526,12 @@ void Display::update_battery(uint8_t r)
 {
     char msg [DISPLAY_MESSAGE_SIZE];
     AP_BattMonitor &battery = AP::battery();
-    uint8_t pct = battery.capacity_remaining_pct();
-    snprintf(msg, DISPLAY_MESSAGE_SIZE, "BAT:%4.2fV %2d%% ", (double)battery.voltage(), pct) ;
+    int8_t pct = -1;
```

Comment:
```suggestion
    int8_t pct;
```

If the method returns true this must have been filled in.

Reply:
right, fixed it.